index.js

```diff
@@ -41,8 +42,8 @@ async function run() {
           CHECK_STATUS_RETRIES,
           CHECK_STATUS_TIME_OUT
         );
-      case "update-clubhouse-workflow":
-        const storyIds = core.getInput("clubhouse-story-ids");
+      case "update-shortcut-workflow":
+        const storyIds = core.getInput("shortcut-story-ids");
```

Comment:
we also need to update these in `main.yml`

Reply:
sure https://github.com/rotabull/rotabull/pull/2068/commits/51aca506c4fdcbf18503b2549ad176e95f278099 good catch 👨